daemon/container.go

```diff
@@ -1036,6 +1036,9 @@ func (container *Container) networkMounts() []execdriver.Mount {
 }
 
 func (container *Container) addLocalMountPoint(name, destination string, rw bool) {
+	if container.MountPoints == nil {
+		container.MountPoints = map[string]*mountPoint{}
```

Comment:
@LK4D4 I think that's what @ibuildthecloud suggested in https://github.com/docker/docker/issues/13435#issuecomment-104982260 but @coolljt0725 was trying to avoid allocating memory until needed: https://github.com/docker/docker/pull/13448#issuecomment-105153020